runtime/browser/web_view_impl.cc

```diff
@@ -100,12 +100,13 @@ void WebViewImpl::LoadUrl(const std::string& url, const std::string& mime) {
   SCOPE_PROFILE();
   if (!mime.empty()) {
     mime_ = mime;
+    url_ = url;
     auto mime_override_cb = [](const char* url, const char* mime,
                                char** new_mime, void* data) -> Eina_Bool{
       WebViewImpl* view = static_cast<WebViewImpl*>(data);
       if (view != nullptr &&
           common::utils::BaseName(url) ==
-            common::utils::BaseName(view->GetUrl())) {
+            common::utils::BaseName(view->url())) {
```

Comment:
url_ is used in only this line. why don't you use url parameter directly instead of adding url() setter/getter.